src/SendGrid/Reliability/ReliabilitySettings.cs

```diff
@@ -64,10 +64,10 @@ public ReliabilitySettings(int maximumNumberOfRetries, TimeSpan minimumBackoff,
                 throw new ArgumentOutOfRangeException(nameof(minimumBackoff), "minimumBackoff must be less than maximumBackOff");
             }
 
-            this.MaximumNumberOfRetries = maximumNumberOfRetries;
```

Comment:
I personally agree with these changes, but the `this.` prefix is the convention used throughout the entire solution. I think following the conventions here to keep the code consistent is more valuable than removing a redundant identifier.

Reply:
I have another PR, which removes `this` thorughtout the codebase